### ISSUE TITLE
Timer conversion in 1.5: Throttle Trigger

### ISSUE THREAD

**Issue by Dsrenger** [FrSkyRC/ETHOS-Feedback-Community#3576]
I just updated one of my transmitters from 1.4.15 to 1.5.1

1.4.15 I usually have my flight time timer set up like this:

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/51758752/92bf4692-8a1e-495c-9ed7-3ac7abacf4f3)

With that, the Timer starts with the throttle, but never stops running, until i reset it.

This timer gets converted to the following:

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/51758752/3ab737e6-4396-41d0-aad9-a30af33f1951)

To get the same functionality it should get converted to:

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/51758752/498e3fdd-8836-4280-b520-17044d31aee9)

So basically, as there is no "Throttle trigger" anymore the throttle trigger should convert to "Throttle active"  with Stop condition "custom" and stop condition "blank"

This would reduce the need for manual adjustmenst after upgrading.

**Comment by vasyly71** [comment]
I have similar experience (and more) described in #3534

**Comment by raydars** [comment]
I just used the proportional timing source today, I thought it was wonderful.    I set my start and reset condition to my kill switch and after that the proportional timer linked to throttle done the rest.    I am quite new to Ethos so have no historic timers to concern myself about on earlier versions.

**Comment by vasyly71** [comment]
> I just used the proportional timing source today, I thought it was wonderful. I set my start and reset condition to my kill switch and after that the proportional timer linked to throttle done the rest. I am quite new to Ethos so have no historic timers to concern myself about on earlier versions.

@raydars no argument here - it is great and new features and granular control is great. But the problem is firmware update from 1.4.x to 1.5.0 has a side effect on timer behavior for pre-existing models. This can be addressed by paragraph in release notes or a like. But manual procedures often result in human errors, procrastination etc which may result in batteries drained in the air followed by crash (i hope note)

**Comment by bsongis-frsky** [comment]
Conversion issue, the fix is ready for 1.5.2
